com.unity.ml-agents/Runtime/Communicator/UnityRLCapabilities.cs

```diff
@@ -0,0 +1,31 @@
+using UnityEngine;
+
+namespace MLAgents
+{
+    /// <summary>
+    /// A struct holding the capabilities flags for Reinforcement Learning across C# and the Trainer codebase.  This
+    /// struct will be used to inform users if and when they are using C# / Trainer features that are mismatched.
+    /// </summary>
+    internal struct UnityRLCapabilities
+    {
+        internal bool BaseRLCapabilities;
```

Comment:
I think we need to set this (and any other flags that we add later) to true in the constructor.

Reply:
yeah I do that in the construction in the Academy, but it would be cleaner and less error-prone in a default constructor.